packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -153,20 +153,56 @@
 
 				<!-- Variable charts -->
 				<div v-if="!showSpinner" class="form-section">
-					<h5>Variables</h5>
 					<section v-if="modelConfig && csvAsset" ref="outputPanel">
-						<template v-for="(cfg, index) of node.state.chartConfigs" :key="index">
-							<tera-chart-control
-								:chart-config="{ selectedRun: 'fixme', selectedVariable: cfg }"
-								:multi-select="false"
-								:show-remove-button="true"
-								:variables="Object.keys(pyciemssMap)"
-								@configuration-change="chartProxy.configurationChange(index, $event)"
-								@remove="chartProxy.removeChart(index)"
-							/>
-							<vega-chart :are-embed-actions-visible="true" :visualization-spec="preparedCharts[index]" />
+						<h5>Parameters</h5>
+						<tera-chart-control
+							:chart-config="{ selectedRun: 'fixme', selectedVariable: selectedParameters }"
+							:multi-select="true"
+							:show-remove-button="false"
+							:variables="Object.keys(pyciemssMap).filter((c) => modelPartTypesMap[c] === 'parameter')"
+							@configuration-change="updateSelectedParameters"
+						/>
+						<template v-for="param of node.state.selectedParameters" :key="param">
+							<vega-chart
+								:are-embed-actions-visible="true"
+								:visualization-spec="preparedDistributionCharts[param].histogram"
+							>
+								<template v-slot:footer>
+									<table class="distribution-table">
+										<thead>
+											<tr>
+												<th scope="col"></th>
+												<th scope="col">{{ preparedDistributionCharts[param].stat.header[0] }}</th>
+												<th scope="col">{{ preparedDistributionCharts[param].stat.header[1] }}</th>
+											</tr>
+										</thead>
+										<tbody>
+											<tr>
+												<th scope="row">Mean</th>
+												<td>{{ preparedDistributionCharts[param].stat.mean[0] }}</td>
+												<td>{{ preparedDistributionCharts[param].stat.mean[1] }}</td>
+											</tr>
+											<tr>
+												<th scope="row">Variance</th>
+												<td>{{ preparedDistributionCharts[param].stat.variance[0] }}</td>
+												<td>{{ preparedDistributionCharts[param].stat.variance[1] }}</td>
+											</tr>
+										</tbody>
+									</table>
```

Comment:
FYI: No action needed for now, there might be a chance that we need to move the table "into" the chart itself so it can be exported along with the graph. The idea is to better support the notions of reproducibility and transparency.

Reply:
From my experience, drawing table with vegalite was limited and tricky. But I think we can still explore more if there are any other ways or work arounds moving forward.